trim/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for trim 0.01
+// Project: https://www.npmjs.com/package/trim
+// Definitions by: Steve Jenkins <https://github.com/skysteve>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "trim" {
```

Comment:
could you strip `declare module`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred